ansible/configs/sap-smart/post_software.yml

```diff
@@ -133,6 +133,28 @@
         mode: '0600'
         force: yes
 
+- name: Prevent Pulp 500 error
+  hosts: sats
+  become: yes
+  tasks:
+
+    - name: Make sure MongoDB will be able to start
+      shell: >
+        echo "-vv" > /.autorelabel &&
+        foreman-maintain service stop &&
+
+    - name: Reboot Satellite server
+      shell: "sleep 5 && reboot"
+      async: 1
+      poll: 0
```

Comment:
Perhaps use `pause` instead of sleeping in a shell command? https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pause_module.html

And rather than using the `reboot` command, use the `reboot` module?
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/reboot_module.html